### PR TITLE
Add the `csi-addons` executable for admins and testers

### PR DESCRIPTION
**Pull request by nixpanic**
The `csi-addons` executable can be used to call CSI-Addons operations against a
CSI-driver. It is included in the side-car container image, so that manual
execution by admins and (automated) testing can easily be done.

### CODE REVIEW

**Review by yati1998 — COMMENTED** [review]
LGMT

**Comment by nixpanic** [comment]
Addressed the merge conflict. This is ready for inclusion IMHO :)

Manually tested with the container-image from `quay.io/nixpanic/csi-addons-k8s-sidecar:cmd_csi-addons`.

**Review by yati1998 — DISMISSED** [review]

**Review by Yuggupta27 — APPROVED** [review]
LGTM! Thanks!

**Comment by nixpanic** [comment]
yet an other merge conflict addressed!

**Review by Yuggupta27 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -70,6 +70,7 @@ check-all-committed: ## Fail in case there are uncommitted changes
 .PHONY: build
 build: generate fmt vet ## Build manager binary.
 	go build -o bin/manager cmd/manager/main.go
+	go build -o bin/csi-addons ./cmd/csi-addons
```
Comment (humblec) [inline Makefile:73]: do we need to place this in `tools`/`debug` dir?

Comment (nixpanic) [reply Makefile:73]: Some documentation is included in the directory where the tool has its files? Documentation for executables can probably best be placed alongside their `main.go` file.

**Review by Rakshith-R — APPROVED** [review]

**Pull request merged**